docs/user-documentation/resource-nodes.md

```diff
@@ -104,19 +107,19 @@ the "Members" tab, Actions can be performed in bulk using the checkboxes and Act
 
 See [the media section](media.md) for more details.
 
-## Display Hints
+## Display Modes
 
-Drupal uses "View Modes" to provide alternative ways to present content to users. You may be familiar with the "full" and "teaser" versions of nodes, which are rendered using two corresponding kinds of View modes. Islandora also uses view modes to control how media content is displayed. Islandora provides two view modes for Media, one which renders the OpenSeadragon viewer and the other which renders the PDFjs viewer. These two View modes can be set explicitly in the node edit form or you can configure Islandora to use a specific View mode for all media with a specific Mime type using [Contexts](context.md).
+Drupal uses "display modes" (sometimes called "view modes") to provide alternative ways to present content to users. You may be familiar with the "full" and "teaser" versions of nodes, which are rendered using two corresponding display modes. Islandora makes use of display modes to control how media content is displayed. Islandora Defaults provides two view modes for Media, one which renders the OpenSeadragon viewer and the other which renders the pdf.js viewer. These two view modes can be enabled by using "Display hints" in the node edit form, or you can configure Islandora to use a specific display mode for all media based on the file's Mime type. Both methods make use of [Contexts](context.md).
```

Comment:
IIRC, "view modes" is the parent term that includes both "form modes" and "display modes".

Reply:
Yes. That was a terrible decision, IMO, because View modes (especially with a capital V) naturally brings to mind the different modes that Views have. I mean, they're technically called "displays". But that term is rarely if ever in the GUI.

The things that we're talking about - the view modes of entities - are those things in the GUI most often accessed under "Manage Display". I mean, sure there's "form display" and "display".  The word "Display" is therefore closely tied to this concept, more than "View". 

I hope that (sometimes called "view modes") is okay as my edit of this. The concept of "Display Modes" or "View Modes" needs a much larger explanation but that does NOT belong in the how-to of switching the way that nodes render in Islandora Defaults, which itself should not be in the conceptual "what is a resource node" page.

Reply:
Yes, going into the nuances between these is out of scope, but I think updating the parenthetical statement to be "a type of 'view mode'" and replace the other instances of "view mode" with "display mode" would be clearer.